src/legacy/state/swap/hooks.tsx

```diff
@@ -83,6 +83,8 @@ interface DerivedSwapInfo {
   currenciesIds: { [field in Field]?: string | null }
   currencyBalances: { [field in Field]?: CurrencyAmount<Currency> }
   parsedAmount: CurrencyAmount<Currency> | undefined
+  // TODO: remove duplications of the value (v2Trade?.maximumAmountIn(allowedSlippage))
+  sellAmountMaxSpend: CurrencyAmount<Currency> | null
```

Comment:
Are we sure we want this and not the allowance?

Reply:
Sorry, I don't understand. Why do wee need allowance here?